Add ASA Entitlements Documentation (#1514)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/asa/entitlements/index.md

* [Update a single sudo Entitlement](#update-a-single-sudo-entitlement)

### List the sudo Entitlements for a team

<ApiOperation method="GET" url="https://app.scaleft.com/v1/teams/${team_name}/entitlements/sudo" />
This endpoint lists the sudo Entitlements for a team.

This endpoint requires one of the following roles: `access_admin`.

#### Request path parameters

| Parameter | Type        | Description   |
| --------- | ----------- | ------------- |
| `team_name`   | string | The name of your Team. |

#### Request query parameters

| Parameter | Type   | Description |
| --------- | ------------- | -------- |
| `count`   |  number | (Optional) The number of objects per page. |
| `descending`   |  boolean | (Optional) The object order. |
| `offset`   |  string | (Optional) The page offset. |
| `prev`   |  boolean | (Optional) The direction of paging. |

#### Request body

This endpoint has no request body.

#### Response body
This endpoint returns a list of objects with the following fields and a `200` code on a successful call.
| Properties | Type        | Description          |
|----------|-------------|----------------------|

> | `add_env`   | array | A list of environment variables to include when running these commands. See [the sudo documentation](https://www.sudo.ws/man/1.8.13/sudoers.man.html#Command_environment) for more information. |

Reviewer: ```suggestion
| `add_env`   | array | A list of environment variables to include when running structured commands. See [the sudo documentation](https://www.sudo.ws/man/1.8.13/sudoers.man.html#Command_environment). |
```

Please do this for all `add_env` table entries.

| `description`   | string | A description of the Entitlement |
| `id`   | string | The UUID of the Entitlement |
| `name`   | string | A name for the Entitlement |
| `opt_no_exec`   | boolean | Whether to allow commands to execute child processes |

> | `opt_no_passwd`   | boolean | Whether or not to require a password when sudo is run (should generally not be used, as ASA accounts don't require a password) |

Reviewer: ```suggestion
| `opt_no_passwd`   | boolean | Whether to require a password when sudo is run (should generally not be used, as ASA accounts don't require a password) |
```
